docs/design/agent-activity-streaming/mockup.html

```
@@ -0,0 +1,788 @@
+<title>IronClaw · Agent Activity & Streaming — Design Mockup</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)